README.md

```diff
@@ -347,6 +411,14 @@ To rebuild the minified JavaScript you must run: `make build`.
 
 During development you can run `make watch` to trigger a build at each change.
 
+### Production build
+
+To build for production (minified files) you must run: `NODE_ENV=production make build`.
```

Comment:
Use the `NODE_ENV` variable directly into the makefile, and update README accordingly.